resources/views/videos/show/sidebar-settings.blade.php

```diff
@@ -52,6 +52,10 @@
                 <div class="sidebar-tab__section">
                     <power-toggle :active="muteVideo" title-off="Mute video" title-on="Unmute video" v-on:on="handleMuteVideo" v-on:off="handleUnmuteVideo">Mute Video</power-toggle>
                 </div>
+
+                <div class="sidebar-tab__section">
+                    <power-toggle :active="singleAnnotation" title-off="Enable Single-Frame Annotation" title-on="Disable Single-Frame Annotation" v-on:on="handleSingleAnnotation" v-on:off="handleDisableSingleAnnotation">Single-Frame Annotation</power-toggle>
```

Comment:
The title is not very helpful. What about "Enable always creating single-frame annotations" (and "Disable ...")?